LICENSE

```diff
@@ -0,0 +1,14 @@
+BSD 3-Clause License
```

Comment:
I don't think this is the correct license for this repo.  The repo uses NS-3 so I think it has to be  GPL license, since this really doesn't work without NS-3.  I think some of the individual files that don't use NS-3 could be marked with a BSD SPDX code, but I think the library as a whole has to be GPL.

Reply:
I sent them a question asking about this, their ns-3 app store slides mention GPLv2 compatible licenses in the checklist on page 21: https://www.nsnam.org/docs/app-store-overview.pdf

Reply:
Are we going to publish this as one or more apps in the NS3 app-store? or have we done so already?

Reply:
Maybe. The ns-3 app store doesn't seem very active.